website/src/pages/plugins.js

```diff
@@ -76,16 +82,18 @@ const SidebarContainer = styled.div`
   padding-right: 10px;
   border-style: solid;
   border-color: #ffffff #efeff5 #ffffff #ffffff ;
+  @media (max-width: 1200px) {
+    display: none;
+  }
 `;
 
-const PluginCard = styled.div`
+const PluginCard = styled.a`
```

Comment:
Hi @foolwc , I noticed that there was a pr to change the tag `<a>` to `<div>` to avoid a descendant warning, so may I ask why we change it back here? Thanks a lot~

Reply:
Previously, only the `icon` and `read more > ` can link to the target. I thought visitors would also want to click to the `title` or `description` to view detail. So I made the whole Card to a link. Could you give me some suggestions?  Thanks a lot.
<img width="311" alt="截屏2021-08-24 上午7 37 16" src="https://user-images.githubusercontent.com/13642747/130532843-94948249-7453-4622-8c82-5e794c28ada6.png">

Reply:
got you, agree with making the whole card as a link. it would be better to use